src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1114,5 +1119,282 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
+        {
+            string v1LogText =
+              @"{
```

Comment:
Nit: In the v2Log below, you push the sample log against the left margin. I like that better. (Saves file size, marginally easier to read, definitely easier to type).

Reply:
Let's just fix this one for now. I'd rather not lengthen the PR. Let's do it opportunistically.

Reply:
OK